ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml

```diff
@@ -82,6 +82,17 @@ objects:
   - kind: SystemGroup
     name: system:unauthenticated
 
+- kind: RoleBinding
+  apiVersion: authorization.openshift.io/v1
+  metadata:
+    name: ${JOB_NAME_SAFE}-admin
+    namespace: ${NAMESPACE}
+  roleRef:
+    name: view
```

Comment:
This isn't admin access

Reply:
yes, thanks - I changed my mind about admin ;) will update title